### PR TITLE
make the linter happy

### PR DESCRIPTION
**Pull request by FelixMo42**

### CODE REVIEW

File: src/nm-service/nm-food-count-input.service.ts [hunk 1]
```diff
@@ -235,7 +235,7 @@ Example:
     ): Promise<string[]> {
         const orgList = (await NmOrgService.getOrgList()).map((a) => ({
             ...a,
-            nameSearchable: a.nameAltList.join(' ') + ' ' + a.name
+            nameSearchable: `${a.nameAltList.join(' ')}${a.name}`
```
Comment (tehChromic) [inline src/nm-service/nm-food-count-input.service.ts]: missing a space here - this will break the count parse

File: src/nm-service/nm-person.service.ts [hunk 2]
```diff
@@ -169,7 +169,7 @@ export class NmPersonService {
                 })
             )
             .then((a) => a.pop());
-        return a != null || [];
+        return a != null ?? [];
```
Comment (tehChromic) [inline src/nm-service/nm-person.service.ts]: this logic is bad. I think it should be a ?? []. But even better the GoogleSpreadsheetsService.rangeGet should always produce a [];

File: src/nm-service/nm-person.service.ts [hunk 3]
```diff
@@ -251,9 +251,8 @@ export class NmPersonService {
         // optionally, get columns that follow
         endCol?: string
     ): string {
-        return `${CORE_PERSON_SHEET}!${ColumnMap[columnName]}${index || ''}${
-            endCol ? `:${endCol}` : ''
-        }`;
+        return `${CORE_PERSON_SHEET}!${ColumnMap[columnName]}${index ?? ''}${endCol ? `:${endCol}` : ''
+            }`;
```
Comment (tehChromic) [inline src/nm-service/nm-person.service.ts:255]: This isn't right. We are also testing if index is zero. In that case we want a blank string. In your case we will get a zero string.

File: src/nm-service/nm-person.service.ts [hunk 4]
```diff
@@ -264,6 +263,6 @@ export class NmPersonService {
         if (!index) {
             throw new Error('must have an index to get a row range name');
         }
-        return `${CORE_PERSON_SHEET}!${ColumnMap[columnName]}${index || ''}`;
+        return `${CORE_PERSON_SHEET}!${ColumnMap[columnName]}${index ?? ''}`;
```
Comment (tehChromic) [inline src/nm-service/nm-person.service.ts]: Same as above.

File: src/service/google-secrets.service.ts [hunk 5]
```diff
@@ -8,6 +8,6 @@ export class GoogleSecretService {
         const [version] = await secretmanagerClient.accessSecretVersion({
             name
         });
-        return JSON.parse(version.payload?.data?.toString() || '{}');
+        return JSON.parse(version.payload?.data?.toString() ?? '{}');
```
Comment (tehChromic) [inline src/service/google-secrets.service.ts]: this is also wrong. a blank string will not evaluate to null or undefined. We want an object in the case of a blank string.

File: src/service/google-spreadsheets.service.ts [hunk 6]
```diff
@@ -69,15 +69,10 @@ export class GoogleSpreadsheetsService {
         };
         const [gspread] = await Gspread;
 
-        try {
-            await gspread.spreadsheets.batchUpdate({
-                spreadsheetId,
```
Comment (tehChromic) [inline src/service/google-spreadsheets.service.ts]: I'm not so sure these are unnecessary try/catch. We want to console.error when we get an error from interacting with Gspread service which hopefully goes to cloudwatch or similar. But we do not want to break the app. So maybe you think these try/catch should be higher up. But they are not right now, so we don't want to crash crabapple if we get an error in a service like this.
